src/translations/de/shopify.php

```diff
@@ -22,12 +22,22 @@
     'shopify plugin loaded' => 'shopify plugin geladen',
     'Shopify Product' => 'Shopify Produkt',
     'None' => 'Kein/e',
-    'Start typing to filter products ...' => 'Tippen um Inhalte zu filtern...',
+    'Start typing to filter ...' => 'Tippen um Inhalte zu filtern...',
     'Show selected only' => 'Nur Ausgewählte anzeigen',
     'Clear selected' => 'Auswahl löschen',
     'Your shopify domain, for example: yourdomain.com or yourshopifydomain.myshopify.com' =>
         'Ihre Shopify-Domain, z.B.: deinedomain.com or deinehopifydomain.myshopify.com',
     'API return limit, default is 50. Maximum is 250.' => 'API Rückgabelimit, Standard ist 50. Maximum ist 250.',
     'The published status for products in the default field. Default is "any". Other choices are: published, unpublished' =>
         'Der Status für Produkte im Einträge-feld. Standard ist "any". Andere mögliche Eingaben sind: published, unpublished',
+    'Api Key' => 'API-Schlüssel',
+    'Password' => 'Passwort',
+    'Shared Secret' => 'Geteiltes Geheimnis',
+    'Hostname' => 'Hostname',
+    'Limit' => 'Grenze',
```

Comment:
limit is the count limit of how many products are returned by the api – right? In this Case `Produkte Limit` might be a better fit.